libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.scss

```
@@ -7,6 +7,7 @@
   h1 {
     font-size: var(--tx-3XL);
     font-weight: bold;
+    margin-top: 0;
```

We should use the variable
```suggestion
    margin-top: var(--pm-0);
```